### PR TITLE
[FIX] Bad Validation on Multisession Send method

### PR DESCRIPTION
**Pull request by chrisdilbert**
When the send_it method fails the validation tries to compare the error message with wrong string, in the fix instead to compare with an string we search if the string “XmlHttpRequestError” is present on the error message.

### CODE REVIEW

File: pos_multi_session/static/src/js/pos_multi_session.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ odoo.define('pos_multi_session', function(require){
             return PosModelSuper.prototype.on_removed_order.apply(this, arguments);
         },
         ms_on_update: function(message, sync_all){
-            this.ms_syncing_in_progress = true; // don't broadcast updates made from this message
+            this.ms_syncing_in_progress = true;
```
Comment (yelizariev) [inline pos_multi_session/static/src/js/pos_multi_session.js:104]: Is there any reason why have you removed the comment?

**Review by GabbasovDinar — COMMENTED** [review]
@yelizariev This method is works

**Comment by yelizariev** [comment]
It seems that #467 already have needed updates

**Comment by KolushovAlexandr** [comment]
This case is already included in the further update, but in a different form

**Pull request closed without merging**